__tests__/fixtures/add/install-strict-all/.npmrc

```diff
@@ -0,0 +1 @@
+yarn-offline-mirror=./mirror-for-offline
```

Comment:
Why a separate file? Let's move this into `.yarnrc` below.

Reply:
Honestly, I just copied `install-no-strict-all` and just added `.yarnrc` :)

Reply:
Maybe actually this is not needed at all?